homepage/scripts/scripts.js

```diff
@@ -129,14 +129,15 @@ const locales = {
 };
 
 const stageDomainsMap = {
-  'www.adobe.com': 'www.stage.adobe.com',
-  'business.adobe.com': 'business.stage.adobe.com',
-  'learning.adobe.com': 'learning.stage.adobe.com',
-  'helpx.adobe.com': 'helpx.stage.adobe.com',
-  'status.adobe.com': 'status.stage.adobe.com',
-  'news.adobe.com': 'news.stage.adobe.com',
-  'blog.adobe.com': 'blog.stage.adobe.com',
-  'developer.adobe.com': 'developer-stage.adobe.com',
+  'www.stage.adobe.com': {
+    'www.adobe.com': 'origin',
+  },
+  '--homepage--adobecom.hlx.live': {
+    'www.adobe.com': 'origin',
+  },
+  '--homepage--adobecom.hlx.page': {
+    'www.adobe.com': 'origin',
+  },
```

Comment:
looks like this PR will not cover all existing domains since it will remove business, learning, helpx, etc. Is this something we all agreed already? (edited)

Reply:
Hi Sean! The map can be adjusted to your testing needs. I've added what I thought to be the core necessary conversions